### PR TITLE
Images on map vis

### PR DESCRIPTION
**Pull request by bdonald25**
#1563

### CODE REVIEW

**Comment by jaypoulz** [comment]
I'm reassigning this review to @stowellm, because he's the one who best understands the migrating goats issue. I didn't see anything that bothered me with the code still, but Mike might catch something I missed.

**Comment by bdonald25** [comment]
I fixed the migrating goat issue.

**Comment by bdonald25** [comment]
Nvm I thought you meant that your reassigning the issue.

**Comment by stowellm** [comment]
Thank you @bdonald25, my goats are safe and settled in Africa, where they may graze and live happily ever after.  No more Greenland migration.

File: app/assets/javascripts/visualizations/highvis/map.coffee [hunk 1]
```diff
@@ -168,11 +168,49 @@ $ ->
...
+            for i in [1...Object.keys(data.metadata).length]
+              if data.metadata[i].dataset_id == dataSetID
+                metaIndex = i
+                break
 
             # Build info window content
             label  = "<div style='font-size:9pt;overflow-x:none;'>"
             label += "<div style='width:100%;text-align:center;color:#{color};'> " +
-              "#{dataPoint[globals.configs.groupById]}</div>"#<br>"
+              "#{dataPoint[globals.configs.groupById]}</div><br>"
```
Comment (stowellm) [inline app/assets/javascripts/visualizations/highvis/map.coffee]: change `<br>` to `</br>` (makes it easier to interpret)

**Comment by stowellm** [comment]
Minor comment, otherwise reviewed.

**Comment by jaypoulz** [comment]
@bdonald25 if you can fix this today, I'll bring it in to the patch for tomorrow

**Comment by bdonald25** [comment]
@jaypoulz It is all set now.

**Pull request merged**